book/tuto-01-getting-started.md

```diff
@@ -48,33 +48,33 @@ Initializing an OpenGL window with glutin can be done using the following steps:
 fn main() {
     use glium::glutin;
 
-    let mut events_loop = glutin::EventsLoop::new();
-    let wb = glutin::WindowBuilder::new();
+    let mut events_loop = glutin::event_loop::EventLoop::new();
```

Comment:
Can you rename the variable name to `event_loop`?

Reply:
done